stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -99,8 +99,24 @@ function versioninfo(io::IO=stdout; verbose::Bool=false)
     if !isempty(Base.GIT_VERSION_INFO.commit_short)
         println(io, "Commit $(Base.GIT_VERSION_INFO.commit_short) ($(Base.GIT_VERSION_INFO.date_string))")
     end
-    if Base.isdebugbuild()
-        println(io, "DEBUG build")
+    official_release = Base.TAGGED_RELEASE_BANNER == "Official https://julialang.org/ release"
```

Comment:
This sounds like a relatively fragile check. We could introduce a separate variable to identify official builds. But maybe that would be overkill.

Reply:
It is brittle, but I feel like setting the exact string `"Official https://julialang.org/ release"` may be a bit more of a barrier/statement than flipping an official build flag

Reply:
Yeah, I agree.  We haven't changed this banner in almost a decade, I don't think it's ever going to change.

Reply:
OK. Do we have a way to check that we don't print this message in official builds. It's not easy to design tests for this kind of feature...

Reply:
We can check on alpha's & rc's given it's set for those
```
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.10.0-alpha1 (2023-07-06)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |
```

Reply:
I've sometimes wondered though whether the trailing `/` should be removed in the url..? Or is it more proper to have it?